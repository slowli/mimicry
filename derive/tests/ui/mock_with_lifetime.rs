use mimicry_derive::Mock;

#[derive(Mock)]
struct WithLifetime<'a> {
    field: &'a (),
}

fn main() {}
