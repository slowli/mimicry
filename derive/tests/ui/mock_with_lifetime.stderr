error: Mock states with lifetimes are not supported
 --> tests/ui/mock_with_lifetime.rs:4:21
  |
4 | struct WithLifetime<'a> {
  |                     ^^
